CovertInfiltration/Src/CovertInfiltration/Classes/UIStackingIcon.uc

```diff
@@ -0,0 +1,68 @@
+//---------------------------------------------------------------------------------------
+//  AUTHOR:  Xymanek
+//  PURPOSE: This is a reusable panel for showing StackedUIIconData. Mainly used for
+//           faction icons.
+//  NOTE:    The (0,0) position of this panel lies at the center of icon (not at top-left
+//           as other UIPanels do). Please account for that when spawining/positioning
+//---------------------------------------------------------------------------------------
+//  WOTCStrategyOverhaul Team
+//---------------------------------------------------------------------------------------
+
+class UIStackingIcon extends UIPanel;
+
+simulated function SetImageStack(StackedUIIconData IconData, optional bool ValidatePath = true)
+{
+	local string ImagePath;
+
+	MC.BeginFunctionOp("SetImageStack");
+	MC.QueueBoolean(IconData.bInvert);
+	
+	foreach IconData.Images(ImagePath)
+	{
+		if (ValidatePath)
+		{
+			ImagePath = class'UIUtilities_Image'.static.ValidateImagePath(ImagePath);
+		}
+
+		MC.QueueString(ImagePath);
+	}
+	
+	MC.EndOp();
+}
+
+////////////////////////////
+/// UIPanel API override ///
+////////////////////////////
+
+simulated function UIPanel SetSize(float NewWidth, float NewHeight)
+{
+	if (Width != NewWidth || Height != NewHeight )
+	{
+		Width = NewWidth;
+		Height = NewHeight;
+
+		MC.BeginFunctionOp("setIconSize");
+		MC.QueueNumber(Width);
+		MC.QueueNumber(Height);
+		MC.EndOp();
```

Comment:
Are you sure that `setIconSize` takes two parameters? 

    function setIconSize(size)
	{
	    var _loc3_ = undefined;
	    var _loc2_ = undefined;
	    this.ImageSize = size;
	    this.HalfImageSize = size / 2;
        // ...

Reply:
I was also really confused, but that's how it's done in `CovertOpsHeaderRowItem`:
```as
   function populateData(strTitle, strColor, bInvertIcon, strFactionImage0, strFactionImage1, strFactionImage2)
   {
      // ....
      this.factionIcon.setIconSize(42,42);
      this.factionIcon.SetImageStack(bInvertIcon,strFactionImage0,strFactionImage1,strFactionImage2);
   }
```
So i decided to roll with it as it is. Do you think that FXS made a mistake and it actually takes only 1 argument?